packages/editor/src/browser/encoding.ts

```diff
@@ -0,0 +1,257 @@
+/********************************************************************************
+ * Copyright (C) 2018 TypeFox and others.
```

Comment:
I may be mistaken, but I don't believe `TypeFox` would be the correct company in your case.

Reply:
Yes, you are right. I failed to notice. So should I change it to something like "Copyright (c) Microsoft Corporation and others. All rights reserved.", since it's copied from VSCode?

Reply:
I believe you can do something similar to other examples present in the repo:

https://github.com/theia-ide/theia/blob/1f54a26e63e0af5125020d2315886500db4e316c/packages/core/src/common/json-schema.ts#L17-L22

Reply:
@vince-fugnitto Hey, thank you! I just revised the two files according to the example. Is it correct? And is there any further modification to be done?

Reply:
Yes the copyrights look great